docs/examples/custom_function/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM icr.io/quantum-public/quantum-serverless-ray-node:0.9.0-py39
```

Comment:
Bump docker image tag to `0.10.1-py310`?

Reply:
and add to update-component-versions action for this, too ?

Reply:
done in https://github.com/Qiskit-Extensions/qiskit-serverless/pull/1297/commits/c3cd804b1dc4fee7d3050dd512683a6d64b15853